client/src/components/screens/PatientCreation/components/FormModal/index.tsx

```diff
@@ -245,7 +245,6 @@ const FormModal : React.FC<Props> = ({
             try {
               const patientBuilder = new PatientBuilder()
                 .withFamily(values.lastname)
-                .withIsFetus(isFetusType)
                 .withGiven(values.firstname)
```

Comment:
This way, both, the mother and the fetus, loses the `isFetus` non? Unless it's added somewhere else?

Reply:
Yes, it's added somewhere else